R/theme-elements.r

```diff
@@ -278,6 +278,8 @@ el_def <- function(class = NULL, inherit = NULL, description = NULL) {
 
   plot.background     = el_def("element_rect", "rect"),
   plot.title          = el_def("element_text", "title"),
+  plot.subtitle       = el_def("element_text", "title"),
+  plot.annotation     = el_def("element_text", "title"),
```

Comment:
that's a good q. if we set it to that by default and let folks play with
`devtools` version of ggplot2, we'll def get community feedback. I'll mod
that too.

On Mon, Mar 14, 2016 at 6:33 PM, Hadley Wickham <notifications@github.com>
wrote:

> In R/theme-elements.r
> <https://github.com/hadley/ggplot2/pull/1582#discussion_r56087991>:
>
> > @@ -278,6 +278,8 @@ el_def <- function(class = NULL, inherit = NULL, description = NULL) {
> >
> >    plot.background     = el_def("element_rect", "rect"),
> >    plot.title          = el_def("element_text", "title"),
> > +  plot.subtitle       = el_def("element_text", "title"),
> > +  plot.annotation     = el_def("element_text", "title"),
>
> Should this default to right alignment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/ggplot2/pull/1582/files#r56087991>.
>